src/lib/y2storage/callbacks/activate_callbacks.rb

```diff
@@ -21,18 +21,27 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
-require "yast"
 require "storage"
+require "y2storage/dialogs/callbacks/activate_luks"
 
 module Y2Storage
-  # class to implement callbacks used during activate in ruby
+  # class to implement callbacks used during activate
   class ActivateCallbacks < Storage::ActivateCallbacks
+    include Yast::Logger
```

Comment:
I just realized this file doesn't follow Ruby (and YaST) conventions. Is named
`lib/y2storage/callbacks/activate_callbacks.rb`, so it should define `Y2Storage::Callbacks::ActivateCallbacks` (which, btw, is redundant). Directory structure is supposed to follow the namespace structure.

So I would suggest

`lib/y2storage/callbacks/activate.rb` -> `Y2Storage::Callbacks::Activate` (I would go for this)

or

`lib/y2storage/activate_callbacks.rb` -> `Y2Storage::ActivateCallbacks` (also valid)

Same applies, of course, to the corresponding tests